README.md

```diff
@@ -1,48 +1,7 @@
-# Payara Tools for VS Code
+# Payara Tools for VSCode
```

Comment:
It **is** VS Code isn't it?
There's a space I'm sure

Reply:
Yeah you're right, fixed